pod/perldelta.pod

```diff
@@ -338,6 +338,9 @@ versions < 3.22 now works again.  This was broken in Perl 5.31.4.
 Building with mingw.org compilers (version 3.4.5 or later) using mingw runtime
 versions >= 3.21 now works (for compilers up to version 5.3.0).
 
+Makefile.mk, and thus support for dmake, has been removed. It is still possible
+to build Perl using nmake (Makefile) and GNU make (GNUmakefile).
+
```

Comment:
"to build Perl on Windows" might be a bit more clear here

Reply:
This entry is in the "Windows" section, but I guess additional clarification won't hurt.